docs/docs/prompts/index.mdx

```diff
@@ -243,7 +243,8 @@ A: You can delete a prompt version using the MLflow UI or Python API:
 import mlflow
 
 # Delete a prompt version
-mlflow.delete_prompt("summarization-prompt", version=2)
+client = mlflow.MlflowClient
```

Comment:
```suggestion
client = mlflow.MlflowClient()
```
Could we detect this? Not critical though

Reply:
Thanks for the catch!